target/linux/lantiq/base-files/etc/board.d/02_network

```diff
@@ -224,6 +224,15 @@ arcadyan,vgv7510kw22-nor|arcadyan,vgv7510kw22-brn)
 		"2:lan:2" "3:lan:1" "4:lan:4" "5:lan:3" "0:wan:5" "6t@eth0"
 	;;
 
+lantiq,vgv952cjw33-e-ir*)
```

Comment:
Please never use wildcards in base-files, but list all affected devices. With wildcards it will be almost impossible to find all locations of a certain device in the future.